docs/rules/sort-collections.md

```diff
@@ -98,13 +98,22 @@ Defaults:
 
 ```json
 [
-	"scripts",
-	"devDependencies",
+	"config",
 	"dependencies",
-	"peerDependencies",
+	"dependenciesMeta",
+	"devDependencies",
+	"exports",
+	"optionalDependencies",
 	"overrides",
-	"config",
-	"exports"
+	"peerDependencies",
+	"peerDependenciesMeta",
+	"pnpm.allowedDeprecatedVersions",
```

Comment:
Hmm, So, I looked these up on the `pnpm` docs, since I wasn't familiar with some of them, and discovered that all of these (including `overrides`) have been moved to the yaml settings file instead of the `package.json`.  So, it's making me question whether we should include these after all?  I don't know, I could go either way.  But since this feature was mostly motivated to support this pattern, perhaps it's still valuable?  What do you think?

Reply:
pnpm of versions <=9, which support the fields I've added, is [>3 times as much](https://github.com/search?q=%22packageManager%22%3A+%22pnpm%409+path%3A%2F%5E.*package.json%24%2F+&type=code) as [pnpm 10](https://github.com/search?q=%22packageManager%22%3A+%22pnpm%4010+path%3A%2F%5E.*package.json%24%2F+&type=code) so I think it's not unreasonable to include them in the default list until old versions die. I'd be fine with either final decision, so perhaps a third opinion is needed 😄

Reply:
Yeah I think it'd be better to leave pnpm-specific additions out of this PR. pnpm is one of many userland "extensions" to the package.json format. Supporting every collection from every popular extension would be a lot of defaults to maintain.

Proposal: let's treat that as a separate change to be discussed in a separate issue?

Reply:
Makes sense - should I not add `resolutions` and `dependenciesMeta` too?

Reply:
Perhaps you've missed my question, I decided to exclude `resolutions` and `dependenciesMeta` from the defaults 🙂